DRAFT.md

```diff
@@ -1497,108 +1472,111 @@ constexpr polymorphic(allocator_arg_t, const Allocator& a,
   pointers.]_
 
 ```c++
-template <class U, class... Ts>
-explicit constexpr polymorphic(in_place_type_t<U>, Ts&&... ts);
+template <class U=T>
+explicit constexpr polymorphic(U&& u);
 ```
 
-17. _Constraints_: `is_base_of_v<T, U>` is `true`. `is_constructible_v<U,
-  Ts...>` is `true`. `is_copy_constructible_v<U>` is `true`.
-  `is_default_constructible_v<allocator_type>` is `true`.
+11. _Constraints_: `is_same_v<remove_cv_ref_t<U>, U>`.
+   `is_same_v<U, polymorphic>` is `false`.
+   `derived_from<U, T>` is `true`.
+   `is_copy_constructible_v<U>` is `true`.
+   `is_constructible_v<U, U>` is `true`.
+   `U` is not a specialization of `in_place_type_t`.
+   `is_default_constructible_v<allocator_type>` is `true`.
 
-18. _Mandates_: `T` is a complete type.
+12. _Mandates_: `T` is a complete type.
 
-19. _Effects_: Constructs an owned object of type `U` with
-    `std​::​forward<Ts>(ts)...` using the allocator `alloc`.
-
-20. _Postconditions_: `*this` is not valueless.
+13. _Effects_: Constructs an owned object of type `U` with
+  `std​::​forward<U>(u)` using the allocator `alloc`.
 
 ```c++
-template <class U, class... Ts>
-explicit constexpr polymorphic(allocator_arg_t, const Allocator& a,
-                               in_place_type_t<U>, Ts&&... ts);
+template <class U=T>
+explicit constexpr polymorphic(allocator_arg_t, const Allocator& a, U&& u);
 ```
 
-21. _Constraints_: `is_base_of_v<T, U>` is `true` and `is_constructible_v<U,
-  Ts...>` is `true` and `is_copy_constructible_v<U>` is `true`.
+14. _Constraints_: `is_same_v<remove_cv_ref_t<U>, U>`.
+   `is_same_v<U, polymorphic>` is `false`.
+   `derived_from<U, T>` is `true`.
+   `is_copy_constructible_v<U>` is `true`.
+   `is_constructible_v<U, U>` is `true`.
+   `U` is not a specialization of `in_place_type_t`.
+   `is_default_constructible_v<allocator_type>` is `true`.
 
-22. _Mandates_: `T` is a complete type.
+15. _Mandates_: `T` is a complete type.
 
-23. _Effects_: `alloc` is direct-non-list-initialized with `a`. Constructs an
-    owned object of type `U` with `std​::​forward<Ts>(ts)...` using the
+16. _Effects_: `alloc` is direct-non-list-initialized with `a`. Constructs
+    an owned object of type `U` with `std​::​forward<U>(u)` using the
     allocator `alloc`.
 
-24. _Postconditions_: `*this` is not valueless.
-
 ```c++
-template <class U>
-explicit constexpr polymorphic(U&& u);
+template <class U, class... Ts>
+explicit constexpr polymorphic(in_place_type_t<U>, Ts&&... ts);
 ```
 
-25. _Constraints_: `is_base_of_v<T, remove_cvref_t<U>>` is `true`.
-   `is_copy_constructible_v<remove_cvref_t<U>>` is `true`.
-   `is_constructible_v<remove_cvref_t<U>, U>` is `true`.
-   `is_same_v<remove_cvref_t<U>, polymorphic>` is `false`.
-   `is_default_constructible_v<allocator_type>` is `true`.
-   `remove_cvref_t<U>` is not a specialization of `in_place_type_t`.
+17. _Constraints_: `is_same_v<remove_cv_ref_t<U>, U>` is true.
```

Comment:
_This_ is where the `is_same_v<remove_cvref_t<U>, U>` is needed :+1:  ... but spelled correctly:

```suggestion
17. _Constraints_: `is_same_v<remove_cvref_t<U>, U>` is true.
```